src/Microsoft.Azure.SignalR.Management/Serialization/JsonObjectSerializerHubProtocol.cs

```diff
@@ -88,8 +88,7 @@ public bool IsVersionSupported(int version)
 
         public bool TryParseMessage(ref ReadOnlySequence<byte> input, IInvocationBinder binder, out HubMessage message)
         {
-            //We don't need reading message with this protocol.
-            throw new NotSupportedException();
+            return new JsonHubProtocol().TryParseMessage(ref input, binder, out message!);
```

Comment:
We should reuse the `ObjectSerializer` here, so that customers can customize how to deserialize the result.